testing/test-nx-utils/project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "test-nx-utils",
+  "$schema": "../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "testing/test-nx-utils/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/esbuild:esbuild",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/testing/test-nx-utils",
+        "main": "testing/test-nx-utils/src/index.ts",
+        "tsConfig": "testing/test-nx-utils/tsconfig.lib.json",
+        "assets": ["testing/test-nx-utils/*.md"],
+        "esbuildConfig": "esbuild.config.js"
+      }
+    },
+    "lint": {
+      "executor": "@nx/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["testing/test-nx-utils/**/*.ts"]
+      }
+    },
+    "unit-test": {
+      "executor": "@nx/vite:test",
+      "options": {
+        "configFile": "testing/test-nx-utils/vite.config.unit.ts"
+      }
+    }
+  },
+  "tags": ["scope:shared", "type:testing"]
```

Comment:
If this intended only for the `nx-plugin`, then `scope:tooling` would be more appropriate. The `.eslintrc.json` would have to tweaked, see https://github.com/code-pushup/cli/pull/779#discussion_r1703857058

Reply:
tweaked!